integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -504,3 +504,47 @@ def test_write_daytime_interval(spark_tmp_path):
             lambda spark, path: spark.read.parquet(path),
             data_path,
             conf=writer_confs)
+
+
+# generate a df with c1 and c2 column have 25 combinations
+def get_25_partitions_df(spark):
```

Comment:
nit: This is a copy/paste. Should we move it to data_gen???

Reply:
Done